specs/clipboard-api/clipboard.idl

```diff
@@ -0,0 +1,450 @@
+    Windows.Foundation.Uri ApplicationListingUri { get; };
+
+    String PackageFamilyName { get; };
+    Windows.Foundation.Uri ContentSourceWebLink { get; };
+    Windows.Foundation.Uri ContentSourceApplicationLink { get; };
+    Windows.Storage.Streams.IRandomAccessStreamReference Square30x30Logo { get; };
+    Windows.UI.Color LogoBackgroundColor { get; };
+
+    String EnterpriseId { get; };
+
+    String ContentSourceUserActivityJson { get; };
+
+    Boolean IsFromRoamingClipboard { get; };
+  }
+
+  /// <summary>
+  /// An acknowledgement by the system that your app is in the process of
+  /// a delay rendering, with the ability to signal the system when done.
+  /// </summary>
+  [contract(ClipboardContract, 1.0)]
+  [interface_name("Microsoft.ProjectReunion.ApplicationModel.IDataProviderDeferral", 4f9a822c-1bb1-4c3e-b908-e202c8b87f4c)]
+  runtimeclass DataProviderDeferral
+  {
+    void Complete();
+  }
+
+  /// <summary>
+  /// Input to a <see cref="DataProviderHandler"/> function for delay-rendering.
+  /// </summary>
+  [contract(ClipboardContract, 1.0)]
+  [interface_name("Microsoft.ProjectReunion.ApplicationModel.IDataProviderRequest", 25f2c71c-832b-4b8e-b9e9-672f098d2770)]
+  runtimeclass DataProviderRequest
+  {
+    String FormatId { get; };
+    Windows.Foundation.DateTime Deadline { get; };
+    DataProviderDeferral GetDeferral();
+    void SetData(Object value);
+  }
+
+  /// <summary>
+  /// Translates strings containing fragments of HTML to and from the special format
+  /// used by <see cref="StandardDataFormats.Html"/>, described here:
+  /// https://docs.microsoft.com/windows/win32/dataxchg/html-clipboard-format
+  /// </summary>
+  [static_name("Microsoft.ProjectReunion.ApplicationModel.IHtmlFormatHelperStatics", cce26a46-4048-4fb4-99c8-b822d03cd526)]
+  static runtimeclass HtmlFormatHelper
+  {
+    static String GetStaticFragment(String htmlFormat);
+    static String CreateHtmlFormat(String htmlFragment);
+  }
+
+  /// <summary>
+  /// Input parameters for <see cref="DataPackage.OperationCompleted"/> event handler functions.
+  /// </summary>
+  [contract(ClipboardContract, 1.0)]
+  [interface_name("Microsoft.ProjectReunion.ApplicationModel.IOperationCompletedEventArgs", a180ab74-fcc7-42f3-a6bd-5d373a28fc8d)]
+  runtimeclass OperationCompletedEventArgs
+  {
+    DataPackageOperation Operation { get; };
+
+    String AcceptedFormatId { get; };
+  }
+
+  /// <summary>
+  /// Static string-valued properties corresponding to data format IDs known
+  /// to the <see cref="DataPackage"/> system.
+  /// </summary>
+  [contract(ClipboardContract, 1.0)]
+  [static_name("Microsoft.ProjectReunion.ApplicationModel.IStandardDataFormatsStatics", 18f4cbf8-64f1-4ba1-9e64-53648a0443a8)]
+  static runtimeclass StandardDataFormats
+  {
+    static String Text { get; };
+    [deprecated("Uri may be altered or unavailable in future releases. Instead, use WebLink or ApplicationLink.", deprecate, ClipboardContract, 1.0)]

```

Comment:
>deprecated [](start = 5, length = 10)

Please remove all deprecated, they are deprecated in windows sdk already no need to carry forward